src/Components/Coins/CoinsButton.css

```diff
@@ -1,6 +1,5 @@
 #coinsButton {
-  width: 80px;
-  height: 40px;
-  background-color: orange;
-  border: 1px solid;
+  height: 38px;
+  width: 78px;
+  background-color: #ffa500;
 }
```

Comment:
please get used to one order of width-height and stick with it, i recommend width and then height.
also - please explain this change, either in code or in this message